package.json

```diff
@@ -33,6 +33,16 @@
           "when": "view == mainView && viewItem == functionalTests && ledgerDevTools.showSelectTestUseCase",
           "group": "inline"
         },
+        {
+          "command": "selectVariant",
```

Comment:
I think this declaration of command can be removed if we don't need the selection icon.

Reply:
Fixed